src/commands/env.rs

```diff
@@ -44,6 +45,15 @@ fn make_symlink(config: &FnmConfig) -> std::path::PathBuf {
     temp_dir
 }
 
+fn dedup_and_merge_path_variable(path: &Path) -> String {
+    let current_path = std::env::var_os("PATH").expect("Can't read PATH env var");
+    let mut split_paths: Vec<_> = std::env::split_paths(&current_path).collect();
+    split_paths.dedup_by(|a, b| a == b || a.to_str().unwrap().contains("fnm_multishells"));
+    split_paths.insert(0, path.to_path_buf());
```

Comment:
if we would collect into a `std::collections::Set`, and then add the pathbuf, we would get dedup for "free", right?
or if it is harder to collect into a set, we can insert and then simply dedup, instead of dedup and then insert?

Reply:
Ok, Here is the code that uses `std::collections::HashSet`. What do you think? 
```rust
let mut split_paths: HashSet<_> = std::env::split_paths(&current_path)
        .filter(|p| !p.to_str().unwrap().contains("fnm_multishells"))
        .collect();
split_paths.insert(path.to_path_buf());
```
I want to dedup it first to remove the `fnm_multishells` path that doesn't same as the value from `FNM_MULTISHELL_PATH`